kubernetes/README.md

```diff
@@ -1,11 +1,11 @@
 # kubernetes.client
-No description provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
+No description provided (generated by Openapi Generator https://github.com/openapitools/openapi-generator)
 
-This Python package is automatically generated by the [Swagger Codegen](https://github.com/swagger-api/swagger-codegen) project:
+This Python package is automatically generated by the [OpenAPI Generator](https://openapi-generator.tech) project:
 
-- API version: v1.14.4
+- API version: v1.14.6
 - Package version: 10.0.0-snapshot
```

Comment:
should it be updated to 11.0.0?

Reply:
it's updated to 11.0.0a1 in later client-generation. See https://github.com/kubernetes-client/python/pull/931/files#diff-52afe03a439cc1bf9f8883b5f62e7004R7